client/state/themes/test/actions.js

```diff
@@ -871,4 +874,144 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
+
+	describe( '#installAndTryAndCustomize', () => {
```

Comment:
We've discussed this before, but would you consider changing this test to only check if `installTheme` and `tryAndCustomizeTheme` are dispatched (much like the `installAndActivate` test does)? I think that's the actual unit we're testing here. In addition, you could add a test just for `tryAndCustomize` that does the spying on `page` and covers the failure case.

(This way, we might get away without mockery and `redux-mock-store`.)